src/main/java/com/synopsys/integration/alert/channel/email/descriptor/EmailGlobalUIConfig.java

```diff
@@ -27,28 +27,24 @@
 
 import org.springframework.stereotype.Component;
 
-import com.synopsys.integration.alert.channel.email.EmailGroupChannel;
 import com.synopsys.integration.alert.common.descriptor.config.field.CheckboxConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.NumberConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.PasswordConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.TextInputConfigField;
-import com.synopsys.integration.alert.common.descriptor.config.ui.DescriptorMetadata;
 import com.synopsys.integration.alert.common.descriptor.config.ui.UIConfig;
-import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
-import com.synopsys.integration.alert.common.enumeration.DescriptorType;
 import com.synopsys.integration.alert.common.enumeration.EmailPropertyKeys;
 import com.synopsys.integration.alert.common.enumeration.FieldGroup;
 
 @Component
 public class EmailGlobalUIConfig extends UIConfig {
 
-    @Override
-    public DescriptorMetadata generateDescriptorMetadata() {
-        return new DescriptorMetadata("Email", "email", EmailGroupChannel.COMPONENT_NAME, DescriptorType.CHANNEL, ConfigContextEnum.GLOBAL, "envelope", setupFields());
+    public EmailGlobalUIConfig() {
+        super("Email", "email", "envelope");
```

Comment:
Encapsulate lowercase email and envelope into static final strings in the descriptor class since they are the URL and font-awesome icon name.